### DISCUSSION TITLE
上传进度100%后就会 一直 显示（后端处理中...），上传不上去！

### DISCUSSION THREAD

**Discussion by XiaoYin320** [AlistGo/alist#413] [Q&A] [answered]
我有点小问题怎么办
onedrive上传比较大的文件时候2g，上传进度100%后就会 一直 显示（后端处理中...），上传不上去！

**Comment by xhofe** [comment, marked as answer]
上传由服务器中转，上传100%后只是你上传到了服务器，后端处理中才是服务器上传到OneDrive，速度取决于你的服务器带宽。

**Reply by XiaoYin320** [reply]
我直接从od官网上传 ，速度很慢，不到1M/s，
用服务器到ondrive能解除限速吗，我上传文件到服务器速度有30M/s

**Reply by xhofe** [reply]
取决于服务器上传到OneDrive的速度，这个我也不知道速度是多少。

**Reply by XiaoYin320** [reply]
好吧，我再想想办法吧，谢谢了

**Reply by weizai408** [reply]
od账号 如果开了代理  不是只有下载走服务器吗  上传也走？  这个转发是用的iptables？

**Reply by xhofe** [reply]
上传开不开代理都走服务器。

**Reply by weizai408** [reply]
用的哪种转发？  iptables吗

**Reply by xhofe** [reply]
和转发无关

**Reply by MiaM1ku** [reply]
您可以尝试一下服务器套warp，到od的速度可以增加

**Reply by imxuebi** [reply]
alist程序能否直接获取onedrive上传链接？这样效率可能会高一点
